models/note.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from "mongoose";
+import { USER, CONTACT, ORGANIZATION, LEAD } from "./refs";
+
+const noteSchema = new mongoose.Schema({
+  attachedTo: { type: mongoose.Schema.Types.ObjectId, required: true },
```

Comment:
Pipedrive's Activities has Linked to in UI, maybe we should use linked for notes here?